compiler/one-cmds/onnx_legalizer.py

```diff
@@ -36,6 +51,21 @@ def reverse_str(s):
 
 
 def parse_tensor_name(name):
+    """Splits tensor name to base part and serial number
+
+    Most of tensor names have have following format: "tensor_123".
+    This  function breaks name into two values: "tensor_" and 123.
+    Tensor names like this: "321" are broken into "" and 321.
+
+    Serial number is used to create unique tensor names using given base name.
+
+    Args:
+        name (str): tensor name
+
+    Returns:
+        tuple of str, int: base name and serial number of tensor
+
```

Comment:
empty string is necessary or can it be removed? (I'm not used to python docs)

Reply:
I'll remove them, looks like I mixed several different formatting styles =/

If I understand it right, python do not have single format, it is just a string with arbitrary data...

However there are several common "styles".

I am trying to comply with [google style](https://google.github.io/styleguide/pyguide.html), because one-cmds integrates with TF and ONNX. ONNX use similar to google formatting, except they do not describe return values and use "Arguments:" instead of "Args:".